packages/docs-site/src/content/docs/network-reference/contract-addresses.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Contract addresses
+description: Network reference page describing various important addresses on Taiko Alethia and Hekla.
+---
+
+## Taiko Alethia contract addresses
+
+You can find the contract addresses of the latest smart contract deployments of the Taiko Alethia Protocol on Ethereum [here](https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/deployments/mainnet-contract-logs-L1.md).
+
+You can find the smart contract addresses of L2 contracts for the Taiko Alethia Protocol on Taiko Alethia [here](https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/deployments/mainnet-contract-logs-L2.md).
+
+## Taiko Hekla contract addresses
+
+You can find the all L1 and L2 contract addresses of the latest smart contract deployments of the Taiko Alethia Protocol [here](https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/deployments/hekla-contract-logs.md).
+
+## Taiko Alethia bootnode addresses
+
+Find the latest bootnodes here in [simple-taiko-node](https://github.com/taikoxyz/simple-taiko-node/blob/v1.16.1/.env.sample).
```

Comment:
imo we can always use main branch here: https://github.com/taikoxyz/simple-taiko-node/blob/main/.env.sample